string query
---
bool object_in_image
